src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs

```diff
@@ -170,6 +170,30 @@ protected static void WaitFor(int milliseconds)
 			pause.WaitOne(milliseconds);
 		}
 
+		/// <summary>
+		/// Returns the first device listed via `adb devices`
+		/// 
+		/// Output is:
+		/// > adb devices
+		/// List of devices attached
+		/// 89RY0AEFA device
+		/// </summary>
+		/// <returns></returns>
+		protected static string GetAttachedDeviceSerial ()
```

Comment:
should this be here or in DeviceTest? 


https://github.com/xamarin/xamarin-android/blob/master/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/DeviceTest.cs#L19